src/app/zap-templates/zcl/data-model/chip/window-covering.xml

```diff
@@ -109,7 +109,7 @@ limitations under the License.
     <!-- Conformance feature LF & PA_LF | [LF] - for now optional -->
     <command source="client" code="0x05" name="GoToLiftPercentage" optional="true" cli="zcl WindowCovering GoToLiftPercentage">
       <description>Go to lift percentage specified</description>
-      <arg name="liftPercentageValue" type="Percent"/>
+      <arg name="liftPercentageValue" type="Percent" optional="true"/>
       <arg name="liftPercent100thsValue" type="Percent100ths" optional="true"/>
```

Comment:
But my understanding is that for _Matter_ the `liftPercent100thsValue` is required, not optional, and the `liftPercentageValue` is not allowed at all.  Since these XMLs are expressing Matter conformance, not generic data model conformance, why are they not matching the spec for Matter?

Or is my understanding wrong?

Reply:
I thought the code has to be compatible for bridging but if you emphasize this is better to simply remove liftPercentageValue and tiltPercentageValue this is ok for me